jquery.xblock.js

```diff
@@ -161,22 +167,23 @@
 
             $('.xblock', root).not('.xblock-initialized').each(function(index, blockDOM) {
                 var initFnName = $(blockDOM).data('init');
-                if (initFnName) {
-                    // Don't fail when the page still contains XModules
-                    if (initFnName === 'XBlockToXModuleShim') {
-                        console.log('Warning: Unsupported XModule JS init', blockDOM);
-                        return;
-                    }
+                if (!initFnName) {
```

Comment:
@Kelketek and I was added it there exactly for OfficeMix to work. It doesn't work now for other reason.